--- nomacs-3.4.1/cmake/Findqpsd.cmake.orig	1970-01-01 01:00:00.000000000 +0100
+++ nomacs-3.4.1/cmake/Findqpsd.cmake	2016-09-10 21:25:00.580715267 +0200
@@ -0,0 +1,16 @@
+# QPSD_FOUND - system has the LIBQPSD library
+# LIBQPSD_INCLUDE_DIRECTORY - the LIBQPSD include directory
+# LIBQPSD_LIBRARY - The libraries needed to use LIBQPSD
+
+if(LIBQPSD_INCLUDE_DIRECTORY AND LIBQPSD_LIBRARY)
+	set(QPSD_FOUND TRUE)
+else()
+	find_path(LIBQPSD_INCLUDE_DIRECTORY NAMES qpsdhandler.h PATH_SUFFIXES qt5)
+	
+	find_library(LIBQPSD_LIBRARY NAMES qpsd5)
+	
+	include(FindPackageHandleStandardArgs)
+	find_package_handle_standard_args(QPSD DEFAULT_MSG LIBQPSD_INCLUDE_DIRECTORY LIBQPSD_LIBRARY)
+	
+	mark_as_advanced(LIBQPSD_INCLUDE_DIRECTORY LIBQPSD_LIBRARY)
+endif(LIBQPSD_INCLUDE_DIRECTORY AND LIBQPSD_LIBRARY)
--- nomacs-3.4.1/CMakeLists.txt.orig	2016-09-10 21:27:15.454047058 +0200
+++ nomacs-3.4.1/CMakeLists.txt	2016-09-10 21:27:26.287380269 +0200
@@ -155,7 +155,7 @@
 	${TIFF_CONFIG_DIR}
 	${QUAZIP_INCLUDE_DIR}
 	${QUAZIP_ZLIB_INCLUDE_DIR}
-	${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/libqpsd
+	${LIBQPSD_INCLUDE_DIRECTORY}
 	${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/drif
 )
 
