--- nomacs-3.4.1/cmake/FindQuaZIP.cmake.orig	2016-09-09 21:46:55.635020115 +0200
+++ nomacs-3.4.1/cmake/FindQuaZIP.cmake	2016-09-09 21:59:03.015011808 +0200
@@ -6,9 +6,9 @@
 if(QUAZIP_INCLUDE_DIRECTORY AND QUAZIP_LIBRARIES)
 	set(QUAZIP_FOUND TRUE)
 else()
-	find_path(QUAZIP_INCLUDE_DIRECTORY NAMES quazip/quazip.h)
+	find_path(QUAZIP_INCLUDE_DIRECTORY NAMES quazip5/quazip.h)
 	
-	find_library(QUAZIP_LIBRARIES NAMES quazip)
+	find_library(QUAZIP_LIBRARIES NAMES quazip5)
 	
 	include(FindPackageHandleStandardArgs)
 	find_package_handle_standard_args(QUAZIP DEFAULT_MSG QUAZIP_INCLUDE_DIRECTORY QUAZIP_LIBRARIES)
--- nomacs-3.4.1/src/DkLoader/DkImageLoader.cpp~	2015-12-17 11:01:42.000000000 +0100
+++ nomacs-3.4.1/src/DkLoader/DkImageLoader.cpp	2016-04-16 18:26:05.820609479 +0200
@@ -74,7 +74,7 @@
 
 // quazip
 #ifdef WITH_QUAZIP
-#include <quazip/JlCompress.h>
+#include <quazip5/JlCompress.h>
 #endif
 
 // opencv
--- nomacs-3.4.1/src/DkLoader/DkBasicLoader.cpp~	2015-12-17 11:01:42.000000000 +0100
+++ nomacs-3.4.1/src/DkLoader/DkBasicLoader.cpp	2016-04-16 18:26:23.390836011 +0200
@@ -51,7 +51,7 @@
 
 // quazip
 #ifdef WITH_QUAZIP
-#include <quazip/JlCompress.h>
+#include <quazip5/JlCompress.h>
 #endif
 
 #ifdef WITH_WEBP
--- nomacs-3.4.1/src/DkLoader/DkImageContainer.cpp~	2015-12-17 11:01:42.000000000 +0100
+++ nomacs-3.4.1/src/DkLoader/DkImageContainer.cpp	2016-04-16 18:26:39.697712795 +0200
@@ -41,7 +41,7 @@
 
 // quazip
 #ifdef WITH_QUAZIP
-#include <quazip/JlCompress.h>
+#include <quazip5/JlCompress.h>
 #endif
 #pragma warning(pop)		// no warnings from includes - end
 
--- nomacs-3.4.1/src/DkGui/DkDialog.cpp.orig	2016-09-09 22:00:28.425010835 +0200
+++ nomacs-3.4.1/src/DkGui/DkDialog.cpp	2016-09-09 22:01:33.561676757 +0200
@@ -92,7 +92,7 @@
 #include <QPrinterInfo>
 // quazip
 #ifdef WITH_QUAZIP
-#include <quazip/JlCompress.h>
+#include <quazip5/JlCompress.h>
 #endif
 
 #pragma warning(pop)		// no warnings from includes - end
